### DISCUSSION TITLE
虛擬機 PVE 學習成果與心得

### DISCUSSION THREAD

**Discussion by JohnnyXiao1024** [1101-GS4719-C/Python-Community-GS4719-C#1517] [General]
前兩個禮拜，我在學校的軟體授權網中找到ESXi的安裝檔
抱持著好奇的態度，就去查了一下如何安裝跟使用方式
詳情請見：[虛擬機 ESXi 學習成果與心得](https://github.com/NCU-GS4719-Python/Python-Community/discussions/1465)
而在尋找ESXi的安裝方式時，也意外發現許多人會使用PVE來做虛擬機的架設
因此又多了這篇

==以下是一些小小的學習成果==

安裝過程
![346147249_222242033875202_881371972947104415_n](https://github.com/NCU-GS4719-Python/Python-Community/assets/125842539/99fa0179-4c5c-4ae0-a370-23608a73bd91)

安裝好了！
![346144744_747500907152199_5860468105387063399_n](https://github.com/NCU-GS4719-Python/Python-Community/assets/125842539/ecb0ab4f-ffbf-4794-903f-4b6cb3bc4314)

老樣子，使用網頁管理虛擬機
![346151600_583015283945442_6903114691002121625_n](https://github.com/NCU-GS4719-Python/Python-Community/assets/125842539/af8eeee6-a2cc-45d1-a694-ceb27b49224a)

以上報告，喔對，最近覺得NAS很酷
所以去找了一下，發現白群太貴了QQ
正在積極尋找黑群資源，之後玩得差不多後再上來分享。

**Comment by poweichen00** [comment]
這學期的課我也有學到虛擬機使用資料庫，使用指令去叫出資料，真的都要搞非常久。
